ambry-store/src/main/java/com.github.ambry.store/DiskManager.java

```diff
@@ -80,11 +82,12 @@
     diskSpaceAllocator = new DiskSpaceAllocator(diskManagerConfig.diskManagerEnableSegmentPooling,
         new File(disk.getMountPath(), diskManagerConfig.diskManagerReserveFileDirName),
         diskManagerConfig.diskManagerRequiredSwapSegmentsPerSize, metrics);
+    this.stoppedReplicas = stoppedReplicas == null ? new ArrayList<>() : stoppedReplicas;
```

Comment:
perhaps `Collections.emptyList()` can be used here as well.

Reply:
fixed.